### PR TITLE
Chore: fixes after the call

### PR DESCRIPTION
**Pull request by totfront**
1. Fixed paths to the images (learn the way how to import in React)
2. Fixed the semantics errors
3. Fixed positioning of the container

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
           className="header__logo"
-          src="/src/images/logo_place_header.svg"
+          src={headerLogo}
           width="228px"
           height="32px"
           alt="logo"
         />
-        <div classnames="title">Научиться учиться</div>
-        <div classnames="subtitle">
+        <h1 classnames="title">Научиться учиться</h1>
```
Comment (totfront) [inline src/App.js:16]: Avoid using `div`s
Check the [video](https://www.youtube.com/watch?v=mAoxm-AyNCw) of my friend about it with an examples

File: src/styles/header/header.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 .header {
+    position: relative;
```
Comment (totfront) [inline src/styles/header/header.css:3]: All elements with `position: absolute` need a continued to base on. This issue can help `position: relative`

Check [the official doc](https://www.w3schools.com/css/css_positioning.asp) for more

**Review by LalightKosh — COMMENTED** [review]
I have understood what was done, merged the branches and deleted the develop branch.

**Comment by totfront** [comment]
@LalightKosh keep in mind that the develop branch has been deleted and it is nice to have 2 permanent branches (usually it is main[master] and dev[develop]). Check the [git-flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow#:~:text=Gitflow%20is%20a%20legacy%20Git,software%20development%20and%20DevOps%20practices.) for a better understanding of the principle.

**Pull request merged**